includes/html/forms/create-alert-item.inc.php

```diff
@@ -28,11 +28,8 @@
 $mute = $_POST['mute'];
 $invert = $_POST['invert'];
 $name = strip_tages($_POST['name']);
-if ($_POST['proc'] != '') {
-    $proc = $_POST['proc'];
-} else {
-    $proc = '';
-}
+$notes = isset($_POST['$notes']) ? $_POST['$notes'] : '';
```

Comment:
We probably need strip tags here:
`$notes = isset($_POST['$notes']) ? strip_tags($_POST['$notes']) : '';`